src/engine/command.cpp

```diff
@@ -2731,6 +2734,13 @@ void explodelist(const char *s, vector<char *> &elems, int limit)
         elems.add(newstring(start, end-start));
 }
 
+void explodelist(const char *s, std::vector<std::string> &elems, int limit)
```

Comment:
Why do you think I know what's the intended purpose? I identified this code to be the source of memory leaks, and rewrote it so that it would prevent some of the leaks (by using STL), that's all.

Reply:
Maybe we should document *that* then in a docstring. It'd be helpful to document the bits we do not (yet) understand, too.